UPGRADE-3.3.md

```diff
@@ -42,6 +42,16 @@ HttpKernel
  * The `Psr6CacheClearer::addPool()` method has been deprecated. Pass an array of pools indexed
    by name to the constructor instead.
 
+Process
+-------
+
+ * On Windows, `!VAR!` expansion inside escaped arguments is deprecated.
+
+ * Not-inheriting environment variables is deprecated.
+
+ * Configuring `proc_open()` options, Windows compatibility and sigchild
+   compatibility is deprecated - they will be always enabled in 4.0.
```

Comment:
should be split in 2 entries, as ``they will be always enabled in 4.0`` does not apply to options